docs/fastdds/discovery/simple.rst

```diff
@@ -123,11 +123,11 @@ to a multicast address and another one linked to a unicast address.
 *Fast DDS* allows for the configuration of an initial peers list which contains one or more such IP-port address
 pairs corresponding to remote DomainParticipants PDP discovery listening resources, so that the local
 DomainParticipant will not only send its PDP traffic to the default multicast address-port specified by its domain,
-but also to all the IP-port address pairs specified in the initial peers list.
+or to all the IP-port address pairs specified in the initial peers list.
```

Comment:
pairs corresponding to remote DomainParticipants PDP discovery listening resources, so that the local
DomainParticipant will only send its PDP traffic to the IP-port address pairs specified in the initial peers list.